hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.js

```diff
@@ -1,34 +1,29 @@
 import React from "react";
 import styles from "./TeamCard.module.scss";
-import Paper from "@material-ui/core/Paper";
 import Container from "@material-ui/core/Container";
 import Typography from "@material-ui/core/Typography";
 import Button from "@material-ui/core/Button";
+import TitledPaper from "components/general/TitledPaper/TitledPaper";
 
 const TeamCard = ({ members, teamCode, handleEditTeam }) => {
     const title = teamCode === "" ? "Team" : "Team " + teamCode;
 
     return (
-        <>
-            <Typography variant="h2" noWrap>
-                {title}
-            </Typography>
-            <Paper elevation={3} className={styles.paper} square={true}>
-                {members.map((member, i) => (
-                    <Container className={styles.name} key={i}>
-                        <Typography variant="body2" noWrap>
-                            {member}
-                        </Typography>
-                    </Container>
-                ))}
-
-                <Container className={styles.lastRow}>
-                    <Button color="primary" onClick={handleEditTeam}>
-                        Edit
-                    </Button>
+        <TitledPaper title={title}>
+            {members.map((member, i) => (
+                <Container className={styles.name} key={i}>
```

Comment:
Something I learned recently, keys when you map are actually supposed to be globally unique, and this won't be if we did the same thing elsewhere on the dashboard. Something like 
```javascript
key={`team-card-${i}`}
```
should work, assuming it won't complain about there being a space in the key. Or you could use `${member}` in there.

(I know this isn't actually a thing you introduced in this PR, but might as well call it out and fix now)

Reply:
Jokes, I'm wrong about this. Keys just have to be unique amongs siblings, they recommend you don't use the map index for other reasons: https://reactjs.org/docs/lists-and-keys.html#keys

Perhaps making it
```
key={`${member}-${i}`}
```
would be good, since that doesn't just use the map index and gets around the edge case that two people have the same name.